### DISCUSSION TITLE
Would the mod work on Crysis remastered?

### DISCUSSION THREAD

**Discussion by lolmao500** [fholger/crysis_vrmod#4] [General]
If we want to kick this mod up a notch, a VR crysis remastered mod might do it lol. Would it be easy to port the mod to that version once the mod on the original crysis version is done?

Just asking for friends. Thanks a lot for your hard work!

**Comment by fholger** [comment]
Afraid not. Different engine, and they haven't released a modding SDK for it that I could work with.
